src/const/meta.ts

```diff
@@ -1,9 +1,11 @@
-const API_BASE_URL = "https://www.mevblocker.com"
+const API_BASE_URL = "https://www.mevblocker.com";
 
 export const CONFIG = {
-  title: 'MEV Blcoker',
-  description: 'CoW Protocol finds the lowest prices from all decentralized exchanges and DEX aggregators & saves you more with p2p trading and protection from MEV ',
-  descriptionShort: 'The smartest way to trade',
+  title: "MEV Blcoker",
+  description:
+    "CoW Protocol finds the lowest prices from all decentralized exchanges and DEX aggregators & saves you more with p2p trading and protection from MEV ",
+  descriptionShort: "The smartest way to trade",
+  googleAnalyticsID: "G-JV2Q6SXRNR",
```

Comment:
Shouldn't this be from an env var instead of being hard coded?

For example, cow.fi has different keys per env

![image](https://user-images.githubusercontent.com/43217/227177194-566cbdaf-8b35-49ce-89a1-67e2fdcfa7c8.png)